ui/shared/tabs/failureSummary/SuggestionsListItem.jsx

```diff
@@ -110,7 +119,25 @@ export default class SuggestionsListItem extends React.Component {
           >
             <FontAwesomeIcon icon={faBug} title="File bug" />
           </Button>
-          <span>{suggestion.search}</span>
+          <span>{suggestion.search} </span>
+          <Clipboard description="error" text={suggestion.search} />
+          <a
+            href={getLogViewerUrl(
+              selectedJob.id,
+              repoName,
+              suggestion.line_number + 1,
+            )}
+            target="_blank"
+            rel="noopener noreferrer"
+            title="Go to log viewer"
+          >
+            <img
+              alt="Logviewer"
+              src={logviewerIcon}
+              className="logviewer-icon ml-2"
```

Comment:
Let's add `bg-light` to the `className` so that it visually matches the copy button.

Reply:
I have changes the background color to white and bg-light is not needed now as far i think
So, I have not added it